gitCoreJGit/src/main/java/com/virtuslab/gitcore/impl/jgit/GitCoreBranch.java

```diff
@@ -74,13 +74,12 @@ protected RevCommit derivePointedRevCommit() throws GitCoreException {
       ObjectId o = jgitRepo.resolve(getFullName());
       if (o == null) {
         throw new GitCoreNoSuchBranchException(
-            String.format("%s branch \"%s\" does not exist in this repository", branchName,
-                getBranchTypeString(/* capitalized */ true)));
+            "${getBranchTypeString(/* capitalized */ true)} branch '${branchName}' does not exist in this repository");
       }
       c = rw.parseCommit(o);
     } catch (MissingObjectException | IncorrectObjectTypeException e) {
-      throw new GitCoreNoSuchCommitException(String.format(
-          "Commit pointed by %s branch \"%s\" does not exist in this repository", branchName, getBranchTypeString()));
```

Comment:
Less obvious advantage of interpolation compared to format strings: the params were flipped here, that was hard to notice with format string, but instantly obvious with interpolation